File: projectEuler/cLang/prob4.c
#include <stdio.h>
//largest palindrome made from the product of two 3-digit numbers

/*
 *I want to build this as multiple functions.  One should check if my current
 * number is a palindrome.  Another should run a nested for loop to generate
 * all products of 2 three digit numbers.
 *
 */

int checkPal(int number)
{
    int i;
    int thousands, hundreds, tens, ones;
    int segmentedNumber[4];
    for(i = 0; i < 4; i++){
        segmentedNumber = number %
    }
}


int runLoop()
{
    int i = 999;
    int j = 999;
    int currentHighest = 0;
    for(i; i > 99; i--)
    {
        for(j; j > 99; j--)
	{
            if(i * j > currentHighest) currentHighest = i * j;           
	}
    }
    return currentHighest;
}

int main()
{
    int 

}

File: projectEuler/python/prob4.py
#largest palendrome made from product of 2 3 digit numbers

#make a list of all products of 2 3 digit numbers
list_of_products = []

def is_palendrome(test_value):
    reversed_string = str(test_value)[::-1]
    return True if reversed_string == str(test_value) else False

for i in range(100, 999):
    for j in range(100, 999):
        iXj = i * j
        if is_palendrome(iXj):
            list_of_products.append(iXj)

list_of_products.sort()


        
print(list_of_products)